src/ferp/util/memoize.test.js

```diff
@@ -0,0 +1,64 @@
+import test from 'ava';
+import { memoizeStore, memoize, getMemoized } from './memoize.js';
+
+test('memoizeStore creates a store', (t) => {
```

Comment:
This test probably isn't great. I don't like testing the instanceof Map, since that implementation could change over time, and creating a store happens in all the other tests.